sklearn/base.py

```diff
@@ -857,6 +857,7 @@ class TransformerMixin(_SetOutputMixin):
 
     def __sklearn_tags__(self):
         tags = super().__sklearn_tags__()
+        tags.estimator_type = "transformer"
```

Comment:
this is gonna break many things, since we have a bunch of estimators that are transformers, and another type.

I had noticed this before, and I think it makes sense to only check for presence of `transform` and/or `fit_transform` to see if something is a transformer.